index.js

```diff
@@ -23,7 +11,15 @@ async function subscribe(topic_name, subscription_name, subscriber, options) {
     },
   };
 
-  const subscription = pubsub.subscription(subscription_name, options || defaultOptions);
+  const topic = pubsub.topic(topic_name);
+  const subscription = topic.subscription(subscription_name, options || defaultOptions);
+  const [exists] = await subscription.exists();
+
+  if (!exists) {
+    logger.info(`Creating subscription ${subscription_name} on topic ${topic_name}`);
+    await subscription.create();
```

Comment:
Should we put this in a try catch block?If create fails?

Reply:
good catch 👍 I will add a try catch block